analyzers/Umbrella/Umbrella.py

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/env python3
+# encoding: utf-8
+import json
+import requests
+from cortexutils.analyzer import Analyzer
+
+class UmbrellaAnalyzer(Analyzer):
+    def __init__(self):
+        Analyzer.__init__(self)
+        self.service = self.get_param('config.service', None, 'Service parameter is missing')
+        self.api_key = self.get_param('config.api_key', None, 'api_key is missing')
+        self.api_secret = self.get_param('config.api_secret', None, 'api_secret is missing')
+        self.organization_id = self.get_param('config.organization_id', None, 'organization_id is missing')
+        self.query_limit = str(self.get_param('config.query_limit', None, 20))
+
+    def umbrella_runreport(self, destination):
+        base_url = "https://reports.api.umbrella.com/v1/organizations"
+        url = "{}/{}/destinations/{}/activity?limit={}".format(base_url,self.organization_id,destination,self.query_limit)
+        try:
+            r = requests.get(url, auth=(self.api_key, self.api_secret))
+            if r.status_code == 200:
+                return json.loads(r.text)
+            else:
+                self.error('API query failed. Check parameters.')
+        except Exception as e:
+            self.unexpectedError(e)
+
+    def summary(self, raw):
+        taxonomies = []
+
+        if len(raw['requests']) > 0:
+            taxonomies.append(self.build_taxonomy(
+                'info',
+                'Umbrella',
+                'Hits',
+                'True'))
```

Comment:
I would agree and that was my initial intention. However, this would never exceed the "max query" parameter that you have to pass to the api. So if your max query is 10 you would never show more that "10" hits even though there could be 100's of 1000's of hits on the particular domain. Thoughts?

Reply:
This check would be more of a "if there are any hits at all then further investigation is required...if no hits, then block (if IOC) and move on to next analyzer/task."

Reply:
Makes sense